api/swagger-spec/v1.json

```diff
@@ -15011,6 +15015,95 @@
      }
     }
    },
+   "v1.Affinity": {
```

Comment:
Thanks for reminding, we do need more discussion about this.

I just copy @davidopp's prior [comment](https://github.com/kubernetes/kubernetes/pull/19758#issuecomment-172286935) here to make some inputs.
> @bgrant0607 should `Affinity` and the related types (like `NodeSelector`) go in v1 or in extensions/v1beta1? If we do the latter, then the v1 `PodSpec` would have a field that is a pointer to a type that lives in v1beta1. The situation we have here is a little different from when we added `LabelSelector`, since in that case all the types that used `LabelSelector` we put in v1beta1. In previous discussions we decided we didn't need to create a v1beta1.PodSpec, and would just AND the old nodeSelector map with the new Affinity stuff, so `PodSpec` will be in v1.